edx_sga/tasks.py

```diff
@@ -143,11 +191,20 @@ def zip_student_submissions(course_id, block_id, locator_unicode, username):
     """
     locator = BlockUsageLocator.from_string(locator_unicode)
     submissions_dir_name = _get_submissions_base_name(username, block_id, course_id)
-    destination_path = _get_dir_path(submissions_dir_name, locator)
-    _remove_existing_artifacts(destination_path)
-    os.mkdir(destination_path)
-    _collect_student_submissions(block_id, course_id, locator, destination_path)
-    _compress_folder(destination_path)
+    relative_destination_path = _get_submissions_dir_path(submissions_dir_name, locator)
+    _remove_existing_artifacts(relative_destination_path)
+
+    if is_local_storage():
+        content_file = ContentFile("submission collect for course {course_id}".format(course_id=course_id))
+        path = os.path.join(DATA_DIR, relative_destination_path)
+        readme_file = os.path.join(path, "readme.txt")
+        default_storage.save(readme_file, content_file)
```

Comment:
Can you get rid of this logic that creates this readme file? I don't think anything in this `if` block is necessary.

Reply:
Done